ppr-ui/src/store/actions/actions-model.ts

```diff
@@ -317,3 +318,10 @@ export const setMhrLocation: ActionIF = ({ commit }, { key, value }): void => {
 export const setCivicAddress: ActionIF = ({ commit }, { key, value }): void => {
   commit('mutateCivicAddress', { key, value })
 }
+
+export const setMhrRegistrationHomeOwnerGroups: ActionIF = (
+  { commit },
+  groups: MhrRegistrationHomeOwnerGroupIF[]
+): void => {
+  commit('mutateMhrHomeOwnerGroups', groups)
+}
```

Comment:
Prettier doing funny things again i think. 
Is there a way to configure it?

Reply:
Yea, i think it goes over the max char per line. I've updated and saved it without formatting for now.